site-structure.json

```diff
@@ -150,6 +189,7 @@
                     { "pageName": "Uploader", "href": "/UploaderAPI.html" },
                     { "pageName": "Undo", "href": "/to-do/UndoAPI.html" },
                     { "pageName": "Table of Contents", "href": "/to-do/TableOfContentsAPI.html" },
+                    { "pageName": "ChangeApplier API", "href": "/ChangeApplierAPI.html" },
```

Comment:
What do you think about dividing the API ToC up into two sections: Framework and Components? I'd put Framework first, mostly since it's shorter...